cloudhub-1/modules/ROOT/Edits/vpn-high-availability.adoc

```diff
@@ -0,0 +1,44 @@
+= Anypoint VPN High Availability
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::vpc-high-availability.adoc,
+
+
+Your VPN connection consists of two VPN tunnels, each terminating in a different Availability Zone, to provide increased availability to your VPC. During a mandatory upgrade, AWS may briefly disable one of the two tunnels of your VPN connection. When you configure your customer gateway, configure both tunnels for redundancy. When one tunnel becomes unavailable (for example, down for maintenance), AWS automatically routes network traffic to the other available tunnel for that specific VPN connection. When AWS performs updates on one VPN tunnel, they set a lower outbound multi-exit discriminator (MED) value on the other tunnel to ensure the failover.  AWS Site-to-Site VPN https://aws.amazon.com/vpn/site-to-site-vpn-sla/[SLA] also requires two VPN endpoints for each connection.  Redundant Anypoint VPN tunnel configuration prevents a significantly higher risk of MuleSoft service downtime during the AWS auto patching window.
+
+To ensure your applications and related operations are tolerant to Anypoint VPN updates, issues, or individual Customer Gateway failures, implement High Availability with redundant Anypoint VPN connections.
+Each Anypoint VPN connection consists of two endpoints, providing High Availability (HA) on the MuleSoft side by default.
+
+
+== Prerequisites
+
+* Sufficient entitlement for each VPN connection.
+* Two VPN endpoints using different public IP addresses.
+* Two VPN connections in Runtime Manager.
+
+== Configure High Availability with Anypoint VPN
+
+The following example shows a High Availability VPN topology using a single Anypoint VPC and two VPN connections.
+
+A MuleSoft Virtual Private Gateway (VGW) supports one Anypoint VPC association, but it supports up to 10 VPN connections.
+You can locate your VPN Gateways in the same data center, or in different physical locations.
+
+To eliminate all single points of failure, the two customer gateways should be in different data centers, and preferably in different cities. Ideally, use your Disaster Recovery location for the second gateway.
+
+image:vpn-ha-topology.png[VPN High Availability Topology]
+
+Use BGP routing to advertise the same routes via VPN-1 and VPN-2. See https://help.mulesoft.com/s/article/Anypoint-VPN-Path-Selection-using-BGP-Routing[Anypoint VPN Path Selection using BGP Routing] for instructions on how to control path selection via the routing protocol.
```

Comment:
If you can, it is better to flip the text of see links so that the user knows what the link will do before seeing the link itself:

For instructions about controlling path selection via the routing protocol, see...